### PR TITLE
(blog)Sustainable open source blog post

### PR DESCRIPTION
**Pull request by hashimwarren**
Sam Bhagwat pledges Gatsby's commitment to open source.

### CODE REVIEW

File: docs/blog/2020-02-11-founding-organizations/index.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+After a number of conversations with other founding organizations who have helped sustain their communities and keep them relevant in changing technological times, the general tenor was sufficient investment was at least 10% of headcount.
+
+### Gatsby’s commitment to investing in open source & sustaining the community
+
+We wrote this piece, among other reasons, to share our deep commitment to open source technology and to the community. We feel that we have a responsibility to keep Gatsby an incredibly healthy and vibrant ecosystem.
+
+What that means to us:
+
+- **Open-source staffing level.** We have 15 people in our roughly 45-person organization—over 30% by headcount, and 50% of our engineering team—writing open-source code and documentation. We anticipate keeping the R&D ratio (50% of our engineering team) roughly constant for the foreseeable future as our organization continues to grow. We’ll continue to report this figure periodically over time.
```
Comment (calcsam) [inline docs/blog/2020-02-11-founding-organizations/index.md]:
```suggestion
- **Open-source staffing level.** We have 25 people in our roughly 55-person organization — over 45% by headcount — writing open-source code and documentation.  We’ll continue to report this figure periodically over time.
```

File: docs/blog/2020-02-11-founding-organizations/index.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
+---
+title: "Founding Organizations: Creating a Company That Sustains Our Open-Source Community"
+date: 2020-02-11
+author: Sam Bhagwat
+excerpt: "Introducing the Community Investment Commitment -  a commitment by founding organizations to have an open source headcount proportion of at least 10% -- in other words, to dedicate at least one-tenth of their organization to observable open source contributions."
```
Comment (calcsam) [inline docs/blog/2020-02-11-founding-organizations/index.md]:
```suggestion
excerpt: "Like governments, open-source founding organizations have a responsibility to their communities: to keep them thriving and on the cutting edge. They must create a successful business model and continue to invest in open source as they grow."
```

File: docs/blog/2020-02-11-founding-organizations/index.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+
+When a project is created outside an established company, a growingly popular way of sustaining it is to form a commercial organization around it, raise venture capital, hire maintainers, and create hosted services to pay the bills.
+
+We’ll term these “founding organizations”—companies founded by the creator or key contributors of an open-source project to support the project. [A growing number of projects](http://oss.cash) are now sustained in this way. We believe this is exciting for software developers the world over, because better software benefits everyone!
+
+Of course, with great power comes great responsibility. Founding organizations are usually viewed as project stewards and as such have special responsibilities to the community that go beyond ordinary corporate responsibilities or even customer-centricity.
+
+We believe founding organizations have a responsibility to provide public goods and infrastructure to their communities to contribute to the community’s health and long-term success.
+
+### Founding organizations\*\* \*\*provide public goods for their communities
```
Comment (ascorbic) [inline docs/blog/2020-02-11-founding-organizations/index.md]: Are these asterisks deliberate?

File: docs/blog/2020-02-11-founding-organizations/index.md [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+
+Other technologies can fall behind. In the website world, both WordPress and Drupal lack core support for version control and modern frontend tooling. In the infrastructure world, Docker exists and is widely used; however, as a project it has been more or less subsumed by Kubernetes, and today most development happens on Kubernetes rather than the container runtime.
+
+We believe founding organizations’ responsibility to the community extends to a sort of social contract to keep the technology up to date:
+
+- When a technology stays at the cutting edge, the founding organization has fulfilled its social contract.
+- Conversely, when a technology begins to fall behind the times, fails to invest in important public goods or keep up with changing development standards, the community should hold the founding organization largely responsible.
+
+### Staying on the cutting edge requires adequate investment from\*\* \*\*founding organizations
```
Comment (ascorbic) [inline docs/blog/2020-02-11-founding-organizations/index.md]: ..and these

**Pull request merged**